lib/link.js

```diff
@@ -146,7 +146,7 @@ Link.prototype._attachReceived = function(attachFrame) {
   this.emit(Link.Attached, this);
   this._resolveAttachPromises(null, this);
 
-  this._checkCredit();
+  this._checkCredit(/* initial = */ true);
```

Comment:
is there any detriment to doing: `this._checkCredit({ initial: true });` here? for the pure sake of readability.